src/plugins/twitter/_base.scss

```diff
@@ -3,33 +3,78 @@
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  */
 
-.wb-disable {
-	.wb-twitter {
-		.twitter-timeline {
-			text-indent: 0 !important;
+.wb-twitter {
+	/* Align the loading icon relative to the center of the Twitter link in browsers that support :has() */
+	&:has( a.twitter-timeline + .twitter-timeline-loading ) {
```

Comment:
Add an exception for the build sasslint error.

Reply:
Unfortunately this doesn't seem to be fixable (see https://github.com/wet-boew/wet-boew/pull/9529#issuecomment-1437329855), but I'll hold off marking this as a resolved until I get around to opening a stylelint PR.

Reply:
Marking this as resolved since #9539 fixed the build error.